extensions/2.0/Khronos/KHR_lights_punctual/schema/light.schema.json

```diff
@@ -19,7 +19,7 @@
         },
         "intensity": {
             "type": "number",
-            "description": "Intensity of the light source. `point` and `spot` lights use luminous intensity in candela (lm/sr) while `directional` lights use illuminance in lux (lm/m^2)",
+            "description": "Intensity of the light source. \"point\" and \"spot\" lights use luminous intensity in candela (lm/sr) while \"directional\" lights use illuminance in lux (lm/m^2)",
```

Comment:
We use markdown syntax in the core spec in a number of places (e.g. [here](https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/schema/accessor.schema.json#L5)). It is useful for wetzel to generate the documentation.

Reply:
Oh wetzel, how could I forget about you?